src/controllers/file.controller.proto.ts

```diff
@@ -0,0 +1,114 @@
+import grpc from "grpc";
+import * as config from './../config'
+
+import { FileServiceClient } from "./../../proto/generated/files_grpc_pb";
+import { File, ReadReq, FileRes, DeleteReq, DeleteRes, CreateReq, UpdateReq } from "./../../proto/generated/files_pb";
+
+export default class FileControllerProto {
+
+    private client: FileServiceClient;
+
+    constructor() {
+        this.client = new FileServiceClient(config.server_url, grpc.credentials.createInsecure());
+    }
+
+    public create(data: any) {
```

Comment:
If you know the fields of data, you should create an interface for its type and use it